src/Tracing/Instrumentation/EventSubscriber/MessageEventSubscriber.php

```diff
@@ -153,4 +163,15 @@ private function getAttributes(Envelope $envelope): array
 
         return $attributes;
     }
+
+    private function getStrategy(Envelope $envelope): string
+    {
+        /** @var StrategyStamp|null $stamp */
+        $stamp = $envelope->last(StrategyStamp::class);
+        if ($stamp) {
+            return $stamp->getStrategy();
+        }
+
+        return StrategyStamp::STRATEGY_PARENT;
```

Comment:
```suggestion
        return StrategyStamp::STRATEGY_LINK;
```

IMO most of the time we dispatch messages to do things in a different process because we don't care about it in the current one.
In which case I would say that we don't care about the associated trace neither.

Reply:
👍